site/content/en/docs/Guides/development-server.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Development Game Server Mode"
+linkTitle: "Development Game Server Mode"
+date: 2019-02-05T01:20:30Z
+weight: 40
+description: >
+  Development mode allows you to run an experimental build of your game server in Agones quickly.
+---
+
+Development mode allows you to host a game server in Agones but without the lifecycle management.
+This means you can run an experimental build of your game server in the Agones environment without the need of packaging and deploying it to a fleet. This allows you to quickly iterate on your game server code while still being able to plugin to your Agones environment.
+
+## Register your server with Agones
+
+To register your development game server you'll need to know the IP address of the machine running it and the port. With that you'll create a game server config like the one below.
+
+```yaml
+apiVersion: "stable.agones.dev/v1alpha1"
+kind: GameServer
+metadata:
+  name: my-development-server
+  annotations:
+    stable.agones.dev/dev-address: "192.1.1.2" # Replace with the IP address of your development game server.
```

Comment:
Let's explicitly call out that this annotation needs to be added - that this is the magic sauce that makes it all happen :+1: